libdevcore/Assertions.h

```diff
@@ -25,7 +25,7 @@
 #pragma once
 
 #include <boost/exception/info.hpp>
-#include <iosfwd>
+#include <iostream>
```

Comment:
Why?

Reply:
Reordering of #includes somewhere (probably in tests) led to error here. `Assertions.h` uses `std::cerr`; `<iosfwd>` is not enough for it